config/routes.rb

```diff
@@ -1,15 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   resources :merchants do
-    resources :items, controller: 'merchant_items'
-    resources :dashboard, controller: 'merchant_dashboard'
+    resources :items, controller: 'merchant_items', only: [:index, :show]
```

Comment:
Awesome! Are you getting any failures on your end?

Reply:
Nope, all green